src/core/trulens/core/database/orm.py

```diff
@@ -151,6 +151,8 @@ class AppDefinition(base):
             _table_base_name: ClassVar[str] = "apps"
 
             app_id = Column(VARCHAR(256), nullable=False, primary_key=True)
+            app_name = Column(VARCHAR(256), nullable=False)
```

Comment:
Will app_id be automatically created by db now?

Reply:
That happens [here](https://github.com/truera/trulens/pull/1349/files#diff-c81bab3f669553c7ef5a6a15a427fcea421ff65fe8086630398dcaa2b33ef03aR163)

Reply:
For existing apps, we should rename app_id to app_version and populate app_name with some dummy value and app_id with the hash

Reply:
for existing apps, app_id becomes app_name and we'd have to populate app_version with a default (`latest`) and set app_id to the hash